### PR TITLE
Allow only building specific apps in yoshi-monorepo

### PR DESCRIPTION
**Pull request by amitdahan**
### 🔦 Summary
Today `yoshi-monorepo build` builds every `app` in the monorepo.
This PR enables passing apps (similar to `yoshi-monorepo start <app-name>`):

```sh
npx yoshi-monorepo build app-a app-b
```

### CODE REVIEW

**Comment by ranyitz** [comment]
So it's `npx yoshi-monorepo build [app-name...]` (optional variadic argument)

**Comment by amitdahan** [comment]
Yep! Should I mention that anywhere?

**Comment by ranyitz** [comment]
@amitdahan 
> Yep! Should I mention that anywhere?

Yes, sorry for not being clear enough.

We do have a help page which is generated when you run `yoshi-monorepo build --help`.

This page show the usage of the command. IDK if there is an official CLI language to describe the API, but from my experience, that would be the way:

```npx yoshi-monorepo build [app-name...]```

the bracets (`[]`) show that it's an optional argument.
and the dots (`...`) show that it's a variadic argument (meaning that there can be more than one)

File: packages/yoshi-flow-monorepo/src/scripts/build.ts [hunk 1]
```diff
@@ -68,6 +69,28 @@ const build: cliCommand = async function(argv, rootConfig, { apps, libs }) {
...
+
+  if (appNames.length) {
+    appNames.forEach(appName => {
+      const pkg = apps.find(pkg => stripOrganization(pkg.name) === appName);
+
+      if (!pkg) {
+        console.log(
+          chalk.red(
+            `Could not find an app with the name of ${appName} to build`,
+          ),
```
[the remark this answers is not in the document]
Comment (amitdahan) [reply packages/yoshi-flow-monorepo/src/scripts/build.ts:82]: image

**Comment by ranyitz** [comment]
Created https://github.com/wix/yoshi/issues/2315

**Pull request merged**